test/e2e/static-oracle.spec.ts

```diff
@@ -80,6 +81,43 @@ contract('StaticOracle', () => {
     await evm.snapshot.revert(snapshotId);
   });
 
+  describe('_getPoolsForTiers', () => {
+    when('sending none fee tiers', () => {
+      then('returns empty array', async () => {
+        expect(await staticOracle.getPoolsForTiers(tokenA.address, tokenB.address, [])).to.be.empty;
+      });
+    });
+    when('sending fee tiers but none have pool', () => {
+      then('returns empty array', async () => {
+        expect(await staticOracle.getPoolsForTiers(tokenA.address, tokenB.address, [500, 3000])).to.be.empty;
+      });
+    });
+    when('sending fee tiers and some have pools', () => {
+      let pools: { [fees: number]: string } = {};
+      given(async () => {
+        pools = await createPoolsWithSupport({
+          tokenA: tokenA.address,
+          tokenB: tokenB.address,
+          feesAndSupportingPeriod: [
+            {
+              fee: FeeAmount.LOW,
+              supportPeriod: false,
+            },
+            {
+              fee: FeeAmount.MEDIUM,
+              supportPeriod: false,
+            },
+          ],
+        });
+      });
+      then('returns the ones that have pools', async () => {
+        expect(await staticOracle.callStatic.getPoolsForTiers(tokenA.address, tokenB.address, [FeeAmount.LOW, FeeAmount.MEDIUM])).to.eql(
```

Comment:
Correct me if I'm wrong, but it looks like you are testing a scenario where **all** the given tiers have pools. I think it would be better if we tested when some (not all) of the given tiers have pools. 

This could be done by passing another `FeeAmount` to `getPoolsForTiers`